components/alert/src/index.scss

```diff
@@ -14,6 +14,10 @@
   @include utrecht-alert__icon;
 }
 
+.utrecht-alert__content {
```

Comment:
I think this added class is missing from the changelog, but should be mentioned.